code/drasil-example/dblpendulum/lib/Drasil/DblPendulum/Body.hs

```diff
@@ -189,8 +189,8 @@ organizationOfDocumentsIntro :: Sentence
 organizationOfDocumentsIntro = foldlSent 
   [atStartNP (the organization), S "of this", phrase document, 
   S "follows the", phrase template, S "for an", getAcc Doc.srs, S "for", 
-  phrase sciCompS, S "proposed by", refS koothoor2013 `S.and_`
-  refS smithLai2005]
+  phrase sciCompS, S "proposed by", refS koothoor2013 `sC` refS smithLai2005 
+  `sC` refS smithEtAl2007 `S.and_` refS smithKoothoor2016]
```

Comment:
Could you make a separate <kbd>[newcomers](https://github.com/JacquesCarette/Drasil/labels/newcomers)</kbd> issue for investigating whether this sentence could be reused between examples?

Reply:
Yeah for sure! The issue that I have created for investigating this is #3410.